File: CSharp/Unit1/02.Operators and Expressions/06.PointInCircle/PointInCircle.cs
/*
 * Write an expression that checks if given point (x,  y) is within a circle K(O, 5).
*/
using System;
class PointInCircle
{
    static void Main()
    {
        int x = int.Parse(Console.ReadLine());
        int y = int.Parse(Console.ReadLine());
        int radius = 5;
        if ((x*x + y*y)<radius*radius)
        {
            Console.WriteLine("Point (" + x + "," + y + ") is in the circle");
        }
        else
        {
            Console.WriteLine("Point (" + x + "," + y + ") is not in the circle");
        }
    }
}

File: Java/IntroJava/02.Operators and Expressions/src/PointInCircle.java
/*
 * Write an expression that checks if given point (x,  y) is within a circle K(O, 5).
*/
import java.util.Scanner;
public class PointInCircle {

	public static void main(String[] args) {
		Scanner input = new Scanner(System.in);
		int x = input.nextInt();
        int y = input.nextInt();
        input.close();
        int radius = 5;
        if ((x*x + y*y)<radius*radius) {
            System.out.println("Point (" + x + "," + y + ") is in the circle");
        }
        else {
        	System.out.println("Point (" + x + "," + y + ") is not in the circle");
        }
	}

}
